modules/ITK/cli/CMakeLists.txt

```diff
@@ -0,0 +1,82 @@
+if(MSVC11) #i.e. Visual Studio 2012
+	# Fix for VS2012 that has _VARIADIC_MAX set to 10. Don't set too high because it increases compiler memory usage / compile-time.
+	add_definitions( -D_VARIADIC_MAX=10 )
+	# Fix for another VS2012 problem: not all TR1 options are automatically detected, therefore we force them here.
+	add_definitions( -D BOOST_HAS_TR1 )
+	add_definitions( -D BOOST_NO_0X_HDR_INITIALIZER_LIST )
+endif()
+
+#the idea here is to have an X.cxx and a X.md where X denotes the cli-command name.
+set( _cli_files
+	statismo-build-shape-model
+)
+
+#search for pandoc to generate man pages or chm help files
+find_program(PANDOC NAMES pandoc)
+mark_as_advanced(PANDOC)
+
+if(PANDOC)
+	message(STATUS "Pandoc found - generating documentation for the CLI tools")
+	#configuring manpage install dir
+	if(WIN32 AND NOT CYGWIN)
+		#put the manuals in the same directory as the binaries on windows
+		set(CLIDOC_INSTALL_DIR ${INSTALL_BIN_DIR})
+		set(CLIDOC_OUTPUT_DIR "${CMAKE_BINARY_DIR}/doc-cli" )
+	else()
+		#use cmake to find the man pages directory on *NIX
+		include(GNUInstallDirs)
```

Comment:
That's an excellent question and the answer is (to my own surprise) yes.

I tried it with /usr/share and /SomeRandomDir as prefix and it respects them both.

![arch pgk ss](https://cloud.githubusercontent.com/assets/10726791/6169159/e5a5390e-b2cb-11e4-8b99-d6363ca1e337.png)